### ISSUE TITLE
No Issue, just Appreciation

### ISSUE THREAD

**Issue by McNnane** [music-discussion/TownOfHost-TheOtherRoles#38]
Idk if you get any notice or validation for the work you and your team are doing in making this mod and working on stabalizing it and fixing bugs, but thank you so much for this mod. I truly have so much fun playing this with my fiance

**Comment by Tommy-XL** [comment]
Yes, I'm also glad that this mod appeared, which has very interesting roles and new features
